Add files via upload

ycZhang2373 opened the pull request:
Machine Learning in R
Yancheng Zhang(xz3157)
Wanling Bai (wb2390)

zcysxy commented:
Please change the file name to snake_case.

> Your project name should be words only and joined with underscores, no white space. Use .Rmd not .rmd. In addition, all letters must be lowercase.

ycZhang2373 commented:
Hi @zcysxy, thank you so much for pointing this out to me. I've changed the name and added a new pull request here: https://github.com/jtr13/cc22tt/pull/121
